pmm-app/src/react-plugins-deps/components/Table/Table.styles.ts

```diff
@@ -0,0 +1,83 @@
+import { css } from 'emotion';
+import { GrafanaTheme } from '@grafana/data';
+import { selectThemeVariant, stylesFactory } from '@grafana/ui';
+
+export const getStyles = stylesFactory((theme: GrafanaTheme) => {
+  const backgroundColor = selectThemeVariant({ light: 'rgb(247, 247, 249)', dark: '#161719' }, theme.type);
+  // @ts-ignore
```

Comment:
Pls don't disable typescript

Reply:
Typescript need to be disabled here because it seems like grafana didn't update interfaces after changing theme.colors structures